src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -13,25 +13,137 @@
 	z-index: 500;
 }
 
-.modal:not(.flyout-dialog) .modal-dialog {
+.modal .btn-secondary {
+	border-style: solid;
+	border-width: 1px;
+}
+.modal.callout-dialog .modal-dialog {
+	width: 452px;
+}
+.modal.callout-dialog.compact .modal-dialog {
+	width: 288px;
+}
+
+.modal:not(.flyout-dialog):not(.callout-dialog) .modal-dialog {
 	margin: auto;
 	width: 640px;
 	height: 480px;
 }
 
+.modal.callout-dialog {
```

Comment:
this is the css for base modal dialog, please do not add the notebook specific styles here.

Reply:
Understood. When I do my follow up PR work, I will be sure to separate out the styles.